### PR TITLE
Added my profile to the README

### PR DESCRIPTION
**Pull request by Jaynil1611**
- Hey, Eddie I have updated the README and added my profile to it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,13 @@ Any questions let us know
       <img src="https://user-images.githubusercontent.com/23638291/88459015-c94dac00-ceb3-11ea-82df-691a23987640.png" />
       <p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/28">(:100: give your vote)</a></p>
     </td>
-    <td></td>
+     <td align="center">
+      <p><a href="https://github.com/Jaynil1611">Jaynil Gaglani</a></p>
+      <img src="https://user-images.githubusercontent.com/48921037/88678363-00b69580-d10c-11ea-902b-58e25c2c799c.PNG" />
+      <p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/36">(:100: give your vote)</a></p>
+    </td>
+    <td align="center">
```
Comment (eddiejaoude) [inline README.md:87]: the extra column is not needed, should be a total of 2 columns per row

File: README.md [hunk 2]
```diff
@@ -99,5 +105,5 @@ Any questions let us know
     </td>
     <td align="center">
     </td>
-  </tr>
+  </tr>
```
Comment (eddiejaoude) [inline README.md:108]: the extra  spaces are not needed here

**Comment by eddiejaoude** [comment]
Looks good 👍 , I left a couple of small comments 🤓

**Comment by Jaynil1611** [comment]
- I have made the changes!
- Please merge it

**Comment by eddiejaoude** [comment]
Thanks @Jaynil1611 , there is still 1 more comment

**Comment by Jaynil1611** [comment]
- Did that one two!

**Comment by eddiejaoude** [comment]
Please check again, it still showing the extra trailing space has been added, I left a comment on the line `108`

**Pull request merged**